packages/db-sqlite/src/createMigration.ts

```diff
@@ -79,7 +79,7 @@ export const createMigration: CreateMigration = async function createMigration(
         .join('\n')
     }
 
-    if (!upSQL?.length && !downSQL?.length && !forceAcceptWarning) {
+    if (!upSQL?.length && !downSQL?.length && !forceAcceptWarning && !skipVerify) {
```

Comment:
Should be `skipEmpty`, and the file should _not_ be created if the flag is passed. Right now, even if this flag is passed, the file is still created.

Reply:
Updated